File: congruences-and-transformations-calculator/Simple-Build/SB-Execute.py
import os

filename = "SB-64"

with open(f"{filename}.html", "w") as f:
    with open("SimpleBuild.html", "r") as sb:
        f.write(sb.read())
    with open("HelpandDiagram.html", "r") as hd:
        f.write(hd.read())

print(f"{filename}.html successfully created!")

File: congruences-and-transformations-calculator/Simple-Build/SB-Execute.rb
require 'fileutils'

filename = 'SB-64'

FileUtils.copy('SimpleBuild.html', "#{filename}.html")
File.open("#{filename}.html", 'a') { |f| f.write(File.read('HelpandDiagram.html')) }

puts "#{filename}.html successfully created!"
